mdoc/Makefile

```diff
@@ -709,7 +717,8 @@ check-doc-tools: 	check-monodocer-since \
 	check-monodocer-cppwinrtUwp\
 	check-monodocer-frameworkalternate-fromold \
 	check-monodocer-frameworkalternate \
-	check-monodocer-embedded-type \
+	check-monodocer-embedded-type \
+	check-ignored-namespace-generic \
```

Comment:
I mentioned in [this comment](https://github.com/mono/api-doc-tools/pull/251#issuecomment-388083113) that we should have a test here to test the efficacy of importing some of these elements, who's `Type` parameter has now changed (for example, a `ref T` parameter .. I didn't see any of those in `TestEcmaDocs.xml`).

The DocXml that is generated by the compiler should have the triple slash docs in the code (so you can verify that it makes it over into the EcmaXML correctly).